L'exécution de votre programme a déclenché une exception java.lang.RuntimeException : Sorry Dave, I cannot let you use backward with an argument in this exercise. Use a loop instead.
 Merci de corriger votre code.
   at plm.runtime1.MethodsReturning.backward (plm.runtime1.MethodsReturning:19)
   at plm.runtime1.MethodsReturning$$anonfun$haveBaggle$1.apply$mcVI$sp (plm.runtime1.MethodsReturning:40)
   at scala.collection.immutable.Range.foreach$mVc$sp (Range.scala:166)
   at plm.runtime1.MethodsReturning.haveBaggle (plm.runtime1.MethodsReturning:33)
   at plm.runtime1.MethodsReturning$$anonfun$run$1.apply$mcVI$sp (plm.runtime1.MethodsReturning:24)
   at scala.collection.immutable.Range.foreach$mVc$sp (Range.scala:166)
   at plm.runtime1.MethodsReturning.run (plm.runtime1.MethodsReturning:23)
   at plm.core.lang.JVMCompiledLang.runEntity (null:-1)
   at plm.core.model.lesson.ExerciseRunner$3.run (null:-1)
   at java.lang.Thread.run (Thread.java:745)
